### PR TITLE
Javascript Loops - Michael Van Doorn

### PR DESCRIPTION
**Pull request by Michaelvxd**

### CODE REVIEW

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
the `repeatWord` function is outputting 1 less than it should

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+
+  while (count < times + 1) {
```
Comment (kurtyazdizadeh) [inline javascript-loops/main.js:36]: that works! 
You could also make `count = 0` or use the `<=` operator

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

File: javascript-loops/main.js (hunk 1 above)
Comment (Michaelvxd) [reply javascript-loops/main.js:36]: Ah yep <= fits the exercise best to the instructions. Updating!

**Pull request merged**